ansible/roles/ocp-workload-cnad-app-modernization/files/image-puller-operator-group.yml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operators.coreos.com/v1
```

Comment:
We could refactor the image puller operator into its own role to make it reusable.